src/main/java/com/work/management/assembler/ProjectAssembler.java

```diff
@@ -0,0 +1,13 @@
+package com.work.management.assembler;
+
+import com.work.management.entity.Projects;
+import com.work.management.resource.ProjectResource;
+import org.springframework.beans.BeanUtils;
+
+public class ProjectAssembler {
```

Comment:
public final class... make this class final... add javadoc before class also..to explain..what it does?
like...Static utility class to convert from .. to ...